libraries/AP_Logger/AP_Logger.h

```diff
@@ -525,6 +527,8 @@ class AP_Logger
     // last time arming failed, for backends
     uint32_t _last_arming_failure_ms;
 
+    bool _once_downloaded = false;
```

Comment:
If you don't need to be aware of the previous value at startup, I don't think the initialization process is necessary.

Reply:
@muramura Right, we don't need this boolean to be initialized all the way in this case. Thanks a lot.